apps/wolfsshd/wolfsshd.c

```diff
@@ -2413,19 +2413,28 @@ static int StartSSHD(int argc, char** argv)
 #ifdef USE_WINDOWS_API
                     unsigned long blocking = 1;
                     if (ioctlsocket(conn->fd, FIONBIO, &blocking)
-                        == SOCKET_ERROR)
-                        err_sys("ioctlsocket failed");
+                        == SOCKET_ERROR) {
+                        WLOG(WS_LOG_DEBUG, "wolfSSH non-fatal error: "
```

Comment:
I think the socket should get cleaned up here and in the other 2 cases. This might leak conn->fd if accept() succeed?